src/Handler/EchoHandler.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\VarDumper\Handler;
+
+use Yiisoft\VarDumper\HandlerInterface;
+use Yiisoft\VarDumper\VarDumper;
+
+final class EchoHandler implements HandlerInterface
```

Comment:
```suggestion
final class HtmlHandler implements HandlerInterface
```

Reply:
Also it's not about html, but about writing to stdout buffer